documentation/en/api-v1-methods.md

```diff
@@ -464,7 +465,8 @@ Response:
   "Err": "string value",
   "Retry": "auto",
   "NBytesReceived": 9,
-  "FastRetrieval": true
+  "FastRetrieval": true,
+  "AnnounceToIPNI": true
```

Comment:
These seem wrong...

Reply:
These are auto-generated by the docs tool

Reply:
We should update the FastRetrieval naming to be aligned with the KeepUnsealedCopy name we are using. @dirkmc - if there's somewhere we are still using FastRetrieval, let's remove it to reduce confusion...

Reply:
Ok we can do that - note though that it will be a breaking change, so we would need to change the Propose Storage Deal Protocol major version (rather than just the minor version). This will require a little more work on our end.

Reply:
Sorry my mistake - these variable names are API methods that are called by the boost client.
I don't believe anyone outside our team is using these API methods so it should be fine to change them. But the naming is unrelated to this PR so I'd suggest we address it in a follow-up PR so that we can merge this one.

Reply:
that works for me, thank you!